### DISCUSSION TITLE
PostgreSQL

### DISCUSSION THREAD

**Discussion by juliacollins** [nsidc/tech-radar#26] [Tools]
Relational database. Used for metadata (edb-model). Not sure of other NSIDC implementations.

**Comment by bruceraup** [comment]
Some other NSIDC databases on PostgreSQL/PostGIS:

- The GLIMS Glacier Database
- Glacier Photograph Collection
- World Glacier Inventory

**Comment by MattF-NSIDC** [comment]
* Hermes
* Sea Ice DB
* The NSIDC GeoServer (PostGIS)
